Sources/ClientRuntime/Identity/IdentityAPI/Identity.swift

```diff
@@ -11,3 +11,13 @@ import Foundation
 public protocol Identity {
     var expiration: Date? { get }
 }
+
+// Identity v. IdentityT v. IdentityType
+// - Identity is the protocol that all identity types must conform to.
+// - IdentityT is the associated type / generic type name used by protocols like IdentityResolver and Signer.
+// - IdentityType is the enum that's used by IdentityResolverConfiguration to return correct type of identity resolver
+//   for the given auth scheme. E.g., SigV4AuthScheme has idType field as .aws. And identityResolver method in SigV4AuthScheme
+//   returns an identity resolver that returns identity of type .aws. 
+public enum IdentityType: CaseIterable {
```

Comment:
Is the `IdentityType` name used in SRA?  If not, I'd perfer to find a type name that doesn't include the word "Type"

Reply:
Changed to `IdentityKind` to avoid using the word "Type"

Reply:
Also, it's not a name used in SRA - SRA, based on Java, uses `IdentityObject::class.java` to determine the type.